ui/src/router/components/form/validation/schema.js

```diff
@@ -454,13 +454,19 @@ const schema = (maxAllowedReplica) => [
   yup.object().shape({
     config: yup.object().shape({
       log_config: yup.object().shape({
-        result_logger_type: yup
-          .mixed()
-          .required("Valid Results Logging type should be selected")
-          .oneOf(
-            ["nop", "bigquery", "kafka"],
-            "Valid Results Logging type should be selected"
-          ),
+        result_logger_type: yup.string().when("$protocol", {
```

Comment:
actually i think this entire chunk (and the old) is not required, as the UI is a dropdown (HTTP) or switch (UPI), other value of inputs will not be passed